Allow to specify which sanity tests to run / skip / enable (#73)
ansible-community/ansible-test-gh-action: README.md

The actual value of `origin-python-version` passed to the [setup-python] action

### `sanity-tests`

Comma-separated list of sanity tests to run. If not present, all applicable tests are run.

### `sanity-skip-tests`

Comma-separated list of sanity tests to skip.

### `sanity-allow-disabled`

> Allow sanity tests to run which are disabled by default.

Reviewer: How about reusing the default value style used at the top of the inputs list in this document?
```suggestion
Allow running sanity tests that are disabled by default.
**(DEFAULT: `false`)**
```
